gcovr/tests/simple1-drive-subst/Makefile

```diff
@@ -0,0 +1,56 @@
+
+GCOVR_TEST_DRIVE ?= Z:
+
+override GCOVR += --verbose
+
+subst:
+ifeq ($(wildcard $(GCOVR_TEST_DRIVE)/*.*),)
+	cmd.exe /C "subst $(GCOVR_TEST_DRIVE) .." || exit 0
+endif
+
+all: subst
+	cd $(GCOVR_TEST_DRIVE)/simple1-drive-subst && $(CXX) -fprofile-arcs -ftest-coverage -fPIC main.cpp -o testcase
+
+run: txt xml html sonarqube json json_summary coveralls
+
+txt: subst
+	cd $(GCOVR_TEST_DRIVE)/simple1-drive-subst && ./testcase
+	# generate actual output
+	cd $(GCOVR_TEST_DRIVE)/simple1-drive-subst && $(GCOVR) -d --txt -o coverage.txt
+
+xml: subst
+	cd $(GCOVR_TEST_DRIVE)/simple1-drive-subst && ./testcase
+	cd $(GCOVR_TEST_DRIVE)/simple1-drive-subst && $(GCOVR) -d -x -o coverage.xml
+
+html: subst
+	cd $(GCOVR_TEST_DRIVE)/simple1-drive-subst && ./testcase
+	# these test cases also cover some CSS-linking permutations
+	cd $(GCOVR_TEST_DRIVE)/simple1-drive-subst && $(GCOVR) --html coverage-summary-includecss.html
+	cd $(GCOVR_TEST_DRIVE)/simple1-drive-subst && $(GCOVR) --html coverage-summary-linkcss.html --no-html-self-contained
+	cd $(GCOVR_TEST_DRIVE)/simple1-drive-subst && $(GCOVR) --html-details coverage-details-includecss.html --html-self-contained
+	cd $(GCOVR_TEST_DRIVE)/simple1-drive-subst && $(GCOVR) --html-details coverage-details-linkcss.html
```

Comment:
Do we expect significant differences depending on whether or not CSS is included? It seems this was copied from another test that *also* happened to cover CSS things as a side effect, but testing the same things here would just slow down the test suite. Similarly, some of the other report formats might not be needed for this test.